packages/usa-footer/src/test/template.html

```diff
@@ -1,98 +1,107 @@
-<footer class="usa-footer usa-footer--big">
-  <div class="grid-container usa-footer__return-to-top">
+<footer class="usa-footer  usa-footer--big ">
+  <div class="usa-footer__container usa-footer__return-to-top">
     <a href="#">Return to top</a>
   </div>
   <div class="usa-footer__primary-section">
     <div class="grid-container">
       <div class="grid-row grid-gap">
-        <div class="tablet:grid-col-8">
-          <nav class="usa-footer__nav" aria-label="Footer navigation">
+        <div class="grid-col-12 tablet:grid-col-8">
+          <nav class="usa-footer__nav" aria-label="Footer navigation,,">
```

Comment:
Heads up, the aria-label has `,,` to avoid a11y issues on the website. Not necessary in this case.